socialhome/streams/templatetags/slurp_json_context.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context(context):
+    """
+    Serializes ``context["json_context"]`` into JSON and generates a JS
+    script to attach it to ``window.context``
+    Args:
+        context: Current view context
+
+    Returns: "<script>window.context = {<context["json_context"]>}</script>"
+
+    """
+    return _slurp_json_context(context, lambda obj: str(obj))
```

Comment:
For classes that can't be implicitely serialized into JSON. I had the case for `LazyBooleanValue` or something like that. I don't remember exactly which variable it is.